codespell_lib/data/dictionary.txt

```diff
@@ -887,6 +887,7 @@ addiotionally->additionally
 addiotions->additions
 additianal->additional
 additianally->additionally
+additiional->additional
```

Comment:
Perhaps add
```
additiion->addition
additiions->additions
additiionally->additionally
```

Reply:
That is a great catch; I have added those as well. Thank you!